toolchain/lower/testdata/function/generic/call_basic.carbon

```diff
@@ -15,7 +15,8 @@ fn H[T:! type](x: T) {
 }
 
 fn G[T:! type](x: T) -> T {
-  // TODO: call crashing.
+  H(x);
+  // TODO: Call crashes, see TODO in FunctionContext::GetValue()
   // H(T);
```

Comment:
What would `H(T);` mean? (T is the type, right - but the H call needs a value, and the `H(x)` above shows that working) Is it just a "crashes instead of error-ing"?

Reply:
First off, yes, it shouldn't crash.
Second, I thought this is allowed. It's strange (and in this case useless), but the function could accept a type as the value and that would be legal. I inferred this from looking at existing test call.carbon that calls F(i32), but let me know if I misunderstood.